src/controls/measure.js

```diff
@@ -399,6 +437,40 @@ const Measure = function Measure({
     document.getElementById(`${viewer.getId()}`).appendChild(markerElement);
   }
 
+  function createRadiusModal(feature) {
+    const title = 'Ange buffert i meter (ex 1000):';
+    const content = `<div>
+                      <input type="number" id="bufferradius">
+                      <button id="bufferradiusBtn">OK</button>
+                    </div>`;
+    const modal = Modal({
+      title,
+      content,
+      target: viewer.getId(),
+      style: 'width: auto;'
+    });
+    const bufferradiusEl = document.getElementById('bufferradius');
+    bufferradiusEl.focus();
+    const bufferradiusBtn = document.getElementById('bufferradiusBtn');
+    bufferradiusBtn.addEventListener('click', (e) => {
+      const radiusVal = bufferradiusEl.value;
+      const radius = parseFloat(radiusVal);
+      if ((!radius && radius !== 0)
+        || (radius <= 0)) {
+        bufferradiusEl.classList.add('unvalidValue');
```

Comment:
Misspelling. Should be '**in**validValue'.

Also, can't see the point of adding this class since it's not used anywhere else, nor is there any CSS stuff implemented for it. Is this a "might come in handy down the line" type thing?

Reply:
Don't remember what the thinking was here so I'll remove it.